### PR TITLE
added sessions to the API

### LINKED ISSUE #18: Add (mentor) sessions table and api
name, description, time, day

### PR DESCRIPTION
**Pull request by howlowck**
closes #18

### CODE REVIEW

File: migrations/20180213214150_create_sessions_table.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+exports.up = (knex, Promise) =>
```
Comment (dwcares) [inline migrations/20180213214150_create_sessions_table.js:1]: sorry, I dont have context into where this file comes from. is it exported from schema, or is manual create. can you speak to naming?

Comment (dwcares) [inline migrations/20180213214150_create_sessions_table.js:1]: what does 'knex' refer to? generated vs a name you selected?

File: seeds/007-sessions.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+const dbName = 'sessions'
```
Comment (dwcares) [inline seeds/007-sessions.js:1]: is this just sample data? is it exported? does it make sense to have this defined here, versus elsewhere?

**Review by dwcares — APPROVED** [review]
I dont know a ton about how you've architected this, but some work on naming could help with handoff.

File: migrations/20180213214150_create_sessions_table.js (hunk 1 above)
Comment (howlowck) [reply migrations/20180213214150_create_sessions_table.js:1]: The file is auto generated by running `npx knex migrate:make file_name`.  As the command suggests, it is generated by `knex`

Comment (howlowck) [reply migrations/20180213214150_create_sessions_table.js:1]: knex is the database query builder library that support multiple relational databases. [Knex Project](http://knexjs.org/).  All the node ORM packages use knex.

File: seeds/007-sessions.js (hunk 2 above)
Comment (howlowck) [reply seeds/007-sessions.js:1]: It's just sample seed data, not meant for production

**Pull request merged**